api/meter.go

```diff
@@ -151,6 +168,13 @@ func (r *relayMeter) loadData(from, to time.Time) error {
 			r.Logger.WithFields(logger.Fields{"error": err}).Warn("Error loading todays usage data")
 			return err
 		}
+
+		todaysLatency, err = r.Backend.TodaysLatency()
+		if err != nil {
+			r.Logger.WithFields(logger.Fields{"error": err}).Warn("Error loading todays latency data")
+			return err
```

Comment:
I think an error in collecting latency data should not block other metrics from being retrieved.

Reply:
No problem. Do you think a log is sufficient in this case? And should I remove the return only from the `TodaysLatency()` call or from `TodaysUsage()` as well?

Reply:
As the func existed already there was an error return for daily usage throwing an error which would have stopped collection of daily usage metrics. So just trying to understand if they should all be consistent or if only a failure of the latency should not stop the rest of the collection?

Reply:
Good point: that is an inconsistency which needs fixing. In general errors in collecting any of the metrics should not have an effect on other metrics' collection.